### PR TITLE
docs: add metal network configuration guide

### PR DESCRIPTION
**Pull request by smira**
Describe the format and operations to get it working.

### CODE REVIEW

File: website/content/v1.5/advanced/metal-network-configuration.md [hunk 1]
```diff
@@ -0,0 +1,430 @@
...
+resolvers: []
+timeServers: []
+```
+
+Every section is optional, so you can configure only the parts you need.
+The format of each section matches the respective network [`*Spec` resource](<{{ relref "../learn-more/networking-resources" }}>) `.spec` part, e.g the `addresses:`
+section matches the `.spec` of `AddressSpec` resource:
+
+```yaml
+# talosctl get addressspecs bond0/10.68.182.1/31 -o yaml | yq .spec                                                                                                                                                                                                                        1471ms  Thu 28 Sep
```
Comment (frezbo) [inline website/content/v1.5/advanced/metal-network-configuration.md:133]: remove the timestamp

Comment (smira) [reply website/content/v1.5/advanced/metal-network-configuration.md]: thanks, removed, I'm blind. I saw the line looked weird, but I didn't see the timestamp :-D

**Review by frezbo — APPROVED** [review]
:cool:

File: website/content/v1.5/advanced/metal-network-configuration.md [hunk 2]
```diff
@@ -0,0 +1,430 @@
...
+      layer: platform
+```
+
+Example, IPv4 route to `10/8` via `10.68.182.0` gateway:
+
+```yaml
+routes:
+    - family: inet4
+      dst: 10.0.0.0/8
+      src: ""
```
Comment (smira) [inline website/content/v1.5/advanced/metal-network-configuration.md:318]: remove for clarity

File: website/content/v1.5/advanced/metal-network-configuration.md [hunk 3]
```diff
@@ -0,0 +1,430 @@
...
+      up: true
+      mtu: 9000
+      layer: platform
+```
+
+The order of the links in the `links:` section is not important.
+
+#### Bonds
+
+For bonded links, there should be a link resource for the bond itself, and a link resource for each enslaving the link:
```
[the remark this answers is not in the document]
Comment (TimJones) [reply website/content/v1.5/advanced/metal-network-configuration.md:207]: `for each enslaved link` and `for each slave link` both work fine here.

**Review by TimJones — APPROVED** [review]

**Comment by TimJones** [comment]
If it was technically available in v1.4, can we also copy the doc to that version?

**Comment by smira** [comment]
> If it was technically available in v1.4, can we also copy the doc to that version?

yep, will do

**Pull request merged**